### ISSUE TITLE
Analog Stick in PSX

### ISSUE THREAD

**Issue by Siushaga** [Swordfish90/Lemuroid#91]
Can't seem to utilize analog stick in PSX games. Tried with my 8bitdo SN30 pro+, as well as the on-screen virtual pad. Also tried with core settings changed for Pad 1 Type from Standard to Analog or Dualshock with restarts of the core in between with no luck. 

I thought it could perhaps be related to "sendLeftStickEventAsDPAD" boolean that some systems seem to rely on for stick events being sent as dpad inputs, but other systems without that like PSP work fine. Would that be because PSP naturally had an analog, whereas PSX controller was dpad only?
https://github.com/Swordfish90/Lemuroid/blob/master/retrograde-app-shared/src/main/java/com/swordfish/lemuroid/lib/library/GameSystem.kt

Sorry I'm not the best at reading through code and understanding the various dependencies and functions working in tandem.

**Comment by Swordfish90** [comment]
Yeah, sadly the sticks do not work out of the box. You need to go into settings and change Pad 1 Type to "dualshock". This cannot be set by default because some game (such as FFVII) require the standard gamepad.

The field you are referring to is only used on system which don't have sticks.

**Comment by Siushaga** [comment]
Thanks for the response and clarification. I do see where having it set to dualshock worked for some games now, so I must have only tested a few that require standard gamepads - sorry about that! Forgot PSX ended up releasing a dualshock controller later on, I always had the original. :)
https://upload.wikimedia.org/wikipedia/commons/thumb/f/f9/PSX-Original-Controller.jpg/1024px-PSX-Original-Controller.jpg
